pkg/plugins/tainttoleration/taint_toleration.go

```diff
@@ -0,0 +1,111 @@
+package tainttoleration
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"reflect"
+
+	clusterapiv1 "open-cluster-management.io/api/cluster/v1"
+	clusterapiv1alpha1 "open-cluster-management.io/api/cluster/v1alpha1"
```

Comment:
`placement` and `placementdecision` API updated to v1beta1, https://github.com/open-cluster-management-io/api/blob/main/cluster/v1beta1/types_placement.go, need to upgrade the version.